brush/src/components/InputModal.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import InputROM from './InputROM'
+import { Modal, ModalActions, ModalTitle } from 'former-kit'
+
+const InputModal = () => {
```

Comment:
I don't like very well the name of this component... Because `InputModal` could be many things. "Input" for what?

What do you think? Maybe something like `InputROMModal` could be better?

Reply:
Agreed! I think InputROMModal is a much better name.